### DISCUSSION TITLE
so-vits-svc-5.0是否跟咱们这个项目相关。

### DISCUSSION THREAD

**Discussion by hertz-pj** [svc-develop-team/so-vits-svc#149] [General]
https://github.com/PlayVoice/so-vits-svc-5.0

**Comment by Miuzarte** [comment]
是一个东西，都是基于3.0改进而来的，应该只是为了避免重名而用了5.0这个名字

**Comment by gjmhmm8** [comment]
谁知道这个5.0效果怎么样

**Reply by bfloat16** [reply]
效果其实还不错，但是没有浅扩散

**Comment by iiallgaii** [comment]
i think so-vits-svc-4.1 is better and the new f0 FCPE is going to be released , can't wait.
